internal/cli/atlas/deployments/start.go

```diff
@@ -126,7 +125,8 @@ func StartBuilder() *cobra.Command {
 	opts := &StartOpts{}
 	cmd := &cobra.Command{
 		Use:     "start <deploymentName>",
-		Short:   "Start a deployment after you stop a machine, it goes into sleep mode, or restarts.",
+		Short:   "Start a deployment.",
```

Comment:
this was too long and only applicable to local development when this works also with atlas